helical/models/caduceus/caduceus_config.py

```diff
@@ -83,6 +82,6 @@ def __init__(
             "model_name": model_name,
             "pooling_strategy": pooling_strategy,
             "batch_size": batch_size,   
-            "device": device,
+            "device": "cuda",
```

Comment:
add comment that this is a strict requirement
`# Caduceus using Triton for mamba-SSM which requires cuda`